apps/sandbox/src/one-time-data-request/OneTimeDataRequestsPage.tsx

```diff
@@ -124,6 +170,23 @@ export const OneTimeDataRequestsPage = () => {
                     dataRequest.push(personaDataRequest)
                   }
 
+                  if (
+                    state.personaProof.enabled ||
+                    state.accountsProof.enabled
+                  ) {
+                    let poo = OneTimeDataRequestBuilder.proofOfOwnership()
+                    dataRequest.push(
+                      state.accountsProof.enabled
+                        ? (state.personaProof.enabled
+                            ? poo.identity(state.personaProof.data.address)
+                            : poo
+                          ).accounts(
```

Comment:
I wonder if I should change implementation of `OneTimeDataRequestBuilder.proofOfOwnership()` so that following is possible.
```
          if (state.personaProof.enabled) {
                    dataRequest.push(
                      OneTimeDataRequestBuilder.proofOfOwnership().identity(
                        state.personaProof.data.address
                      )
                    )
                  }

                  if (state.accountsProof.enabled) {
                    dataRequest.push(
                      OneTimeDataRequestBuilder.proofOfOwnership().accounts(
                        state.accountsProof.data.addresses.filter(Boolean)
                      )
                    )
                  }
                  ```
                  
                  Currently, last item overrides previous one